example/example-dev.html

```diff
@@ -34,6 +34,15 @@
       <div class="ce-example__button" id="saveButton">
         editor.save()
       </div>
+      <div class="ce-example__statusbar">
```

Comment:
i think, all the changes from `example-dev.html` should be included to the `example.html` as well.